packages/storage-plus/src/map.rs

```diff
@@ -237,6 +277,38 @@ mod test {
         );
     }
 
+    #[test]
+    #[cfg(feature = "iterator")]
+    fn range_triple_key() {
```

Comment:
Good test. This is what I want to see for the full-stack usage.

Can you update the TRIPLE to `(&[u8], Int32Key, Int64Key)` and verify this works just as well?

Reply:
Sure.

Reply:
I fixed the prefix implemention in my last commit.
You should be able to use it in the new code.

Reply:
I need to reflect why we cannot satisfy Prefix automatically for anything that implements PrimaryKey - I assume this is a subset of the work.